packages/xo-web/src/xo-app/settings/config/index.js

```diff
@@ -18,7 +18,13 @@ export default class Config extends Component {
   _importConfig = () => {
     this.setState({ importStatus: 'start' }, () =>
       importConfig(this.state.configFile).then(
-        () => this.setState({ configFile: undefined, importStatus: 'end' }),
+        imported => {
+          if (imported !== false) {
```

Comment:
@badrAZ What do you think of this? It's supposed to handle the case where the user closes the modal without importing the config.

Reply:
I aggree :+1: